spring-cloud-task-samples/batch-job/README.adoc

```diff
@@ -1,6 +1,6 @@
 = Spring Batch Job Task
 
-This is a Spring Cloud Task application that executes two simple Spring Batch Jobs.
+This is a Spring Cloud Task application that executions a single Spring Batch Job.
```

Comment:
Hello @DavidMcClatchey,

Can you update this PR such that it says, "executes a single" instead of "executions a single".   Thanks